common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -85,31 +137,16 @@ def list(self, request, *args, **kwargs):
 
     def perform_destroy(self, instance):
         """
-        This method is an override and is called by the DELETE method
+        This method is an override and is called by the destroy method, which is called the a DELETE operation occurs
```

Comment:
%s/which is called the a DELETE operation occurs/which is called when the a DELETE operation occurs